js/private/nodejs_package.bzl

```diff
@@ -152,6 +152,11 @@ def _nodejs_package_impl(ctx):
     return [
         DefaultInfo(files = files, runfiles = runfiles),
         LinkablePackageInfo(package_name = ctx.attr.package_name, files = [output]),
+        # TODO: figure this out why we need this
```

Comment:
you can just comment
"Always assume that packages provide typings, so we don't need to use an action to inspect the package.json#typings field or search for .d.ts files in the package."